Navidux/Sources/Navidux/NavigationController.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  NavigationController.swift
+//  NaviduxTests
+//
+//  Created by Александр Евсеев on 12.10.2022.
+//  Copyright © 2022 red_mad_robot. All rights reserved.
+//
+
+import UIKit
+
+/// NavigationController its abstract object uses for navigation. Its a closest wrapper to UINavigation controller or alternative.
+/// - parameters:
+/// - **screens**: return some representation of screen from UINavigationController.viewController. Difference in track screen in navigation stack.
```

Comment:
You can do notation about properties and methods directly above them. No need to do it for whole protocol

Reply:
move notations to properties where its necessary